data_acquisition/data_acquisition.py

```diff
@@ -0,0 +1,37 @@
+import requests
```

Comment:
You'll probably want a way to manage dependencies like `requests`. I've found [Pipenv](https://pipenv.readthedocs.io/en/latest/) really nice for Rocket 2, but if you don't go with something like that, you should at least provide a [requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files).

Reply:
added pipenv